server/events/comment_parser.go

```diff
@@ -152,6 +152,11 @@ func (e *CommentParser) Parse(rawComment string, vcsHost models.VCSHostType) Com
 		return CommentParseResult{CommentResponse: fmt.Sprintf(DidYouMeanAtlantisComment, e.ExecutableName)}
 	}
 
+	// Warn user about misspelled atlantis word, altantis being the most common
+	if args[0] == "altantis" && e.ExecutableName == "atlantis" {
```

Comment:
This might be an issue since we can now change the vcs executable name. If you had multiple Atlantis on the same repo and you misspelled it with altantis, as you have proposed here, it would trigger all the Atlantis (atlantii?) unexpectedly.

I think what we'd want instead is to use some module and pass in the vcs executable name to get common misspellings, then check to see if arg[0] is within that last 

cc @krrrr38 for thoughts on this since it could conflict with this pr

- #2805

Reply:
Perhaps it's better to instead use a lowercased string comparison check with the executable name? If the comment arg[0] is off by +/- 2 points then we can assume it's a mispelling and attempt to run atlantis.

Reply:
If i'm not mistaken, the `e.ExecutableName == "atlantis"`condition means that spellcheck will only be triggered when we're interacting with the default `atlantis` executable.

of course we can add a method to add a check if any `e.ExecutableName` has some number of letters off, however I think we might get too deep trying to cover all the cases

Reply:
Terraform use https://github.com/agext/levenshtein btw

https://github.com/hashicorp/terraform/tree/fc8fed0047e98b430f648c3659d22be6f8d52642/internal/didyoumean

Reply:
> Terraform use https://github.com/agext/levenshtein btw
> 
> https://github.com/hashicorp/terraform/tree/fc8fed0047e98b430f648c3659d22be6f8d52642/internal/didyoumean

that is very nice, I would try to use that!

Reply:
I put it under https://github.com/runatlantis/atlantis/issues/3069